src/circle.rs

```diff
@@ -19,8 +22,16 @@ impl VelloCircle {
     }
 }
 
-impl VelloVector for VelloCircle {
+impl Vector for VelloCircle {
     fn shape(&self) -> impl kurbo::Shape {
         kurbo::Circle::new(kurbo::Point::default(), self.radius)
     }
+
+    fn border_translation(&self, _time: f64) -> DVec2 {
+        DVec2::ZERO
+    }
+
+    fn border_rotation(&self, _time: f64) -> f64 {
+        0.0
+    }
```

Comment:
the implementation of `border_translation` should be something like:

```rs
// Prevent time from exceeding 1 so it loops over
let time = time % 1.0;
let theta = time * TAU;
DVec2::new(f64::sin(theta), f64::cos(theta)) * self.radius
```

If I am not mistaken...

And then we can use this value to find the rotation too.